src/lender/admin/member.sol

```diff
@@ -30,10 +17,20 @@ contract MemberAdmin is Auth {
     // Admins can manipulate memberlists, but have to be added and can be removed by any ward on the MemberAdmin contract
     mapping (address => uint) public admins;
 
+    event RelyAdmin(address indexed usr);
+    event DenyAdmin(address indexed usr);
+
     modifier admin { require(admins[msg.sender] == 1); _; }
 
-    function relyAdmin(address usr) public auth note { admins[usr] = 1; }
-    function denyAdmin(address usr) public auth note { admins[usr] = 0; }
+    function relyAdmin(address usr) public auth {
```

Comment:
No, the problem was that `note` was previously included through the `tinlake-auth` dependency, but `ds-note` was actually removed from `tinlake-auth` now. So rather than change this to include `ds-note` here now directly, I thought it was cleaner to then just rewrite it properly.